modules/dxp/apps/saml/saml-test/src/testIntegration/java/com/liferay/saml/internal/servlet/filter/test/SamlSameSiteLaxCookiesFilterTest.java

```diff
@@ -69,15 +66,17 @@ public static void setUpClass() {
 			"SAMLResponse", "TEST_SAMLRESPONSE"
 		).build();
 
-		Set<Map.Entry<String, String>> entrySet = _paramsMap.entrySet();
+		StringBundler sb = new StringBundler();
 
-		Stream<Map.Entry<String, String>> stream = entrySet.stream();
+		for (Map.Entry<String, String> entry : _paramsMap.entrySet()) {
+			sb.append(
+				StringBundler.concat(
+					entry.getKey(), "=", entry.getValue(), "&"));
```

Comment:
@W2jx When using StringBundler, please first pay attention and see if you can give it an initial capacity. Then, since you are already using `append`, don't append any concatenated string; just append the parts one by one.